int32[] target_id
int32[] distance_dat
float32[] speed_dat
float32[] distance_x
float32[] distance_y
int32[] magnitude_dat


